app/src/main/java/com/hammer/talkbbokki/presentation/detail/DetailScreen.kt

```diff
@@ -121,53 +99,43 @@ fun DetailHeader(cardFace: CardFace, onBackClick: () -> Unit) {
 fun DetailFlipCard(
     modifier: Modifier = Modifier,
     cardFace: CardFace,
-    onClickToList: () -> Unit,
     item: TopicItem,
+    onClickToList: () -> Unit,
     onClickBookmark: (Boolean) -> Unit
 ) {
     var scale by remember { mutableStateOf(1f) }
     var rotation by remember { mutableStateOf(1f) }
 
-    FlipCard(
-        rotation = rotation,
+    FlipCard(rotation = rotation,
```

Comment:
🚫 **[ktlint]** <argument-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Argument should be on a separate line (unless all arguments can fit a single line)